op-service/sources/types.go

```diff
@@ -36,6 +36,11 @@ type headerInfo struct {
 	*types.Header
 }
 
+// TODO: required to compile
+func (h headerInfo) ParentBeaconRoot() *common.Hash {
+	return nil
+}
+
```

Comment:
Can omit before merge, the fix for this compiler error is in the destination branch